docs/src/exact.md

```diff
@@ -170,16 +170,19 @@ Root 0.707107 of 2x^2 - 1
 
 julia> QQ(C(pi))
 ERROR: unable to convert to a rational number
+[...]
 
 julia> QQ(C(10) ^ C(10^9))
 ERROR: unable to convert to a rational number
+[...]
 ```
 
 To compute arbitrary-precision numerical enclosures, convert to
 `ArbField` or `AcbField`:
 
-```julia
-julia> CC = AcbField(64);
+```jldoctest
```

Comment:
````suggestion
```jldoctest; setup = :(C = CalciumField())
````

Reply:
Alternatively you could modify the `DocTestSetup` block at the top of this file to defined `C`